api/s3headers/headers.go

```diff
@@ -1,42 +1,50 @@
 package s3headers
 
 const (
+	attributePrefix = "s3"
```

Comment:
NeoFS tends to use classic HTTP attribute naming scheme. So this better be `S3-`